tools/ak2-core.sh

```diff
@@ -483,6 +487,9 @@ patch_prop() {
   fi;
 }
 
+# grep_prop <prop name>
+grep_prop() { grep "^$1" "/system/build.prop" | cut -d= -f2; }
```

Comment:
Same comment as #18, I think `system_getprop` is a bit of a more intuitive name.